src/tracks/playabletrack/wavetrack/ui/WaveformView.cpp

```diff
@@ -687,6 +687,7 @@ void DrawClipWaveform(TrackPanelDrawingContext &context, size_t channel,
    const double &t1 = params.t1;
    const double &averagePixelsPerSample = params.averagePixelsPerSample;
    const double &rate = params.rate;
+   const double &stretchRatio = 1.0;
```

Comment:
I assume you change this line in a later commit, but why not make it right in this commit?